sfs/time/source.py

```diff
@@ -4,6 +4,23 @@
 
 .. include:: math-definitions.rst
 
+.. plot::
+    :context: reset
+
+    import matplotlib.pyplot as plt
+    import numpy as np
+    import sfs
+
+    plt.rcParams['figure.figsize'] = 6, 6  # inches
+
+    xs = 1.5, 1, 0  # point source
+    t = np.linalg.norm(xs) / sfs.default.c
```

Comment:
What's the idea behind choosing this time instant?

I understand that it's the time when the wave front crosses the origin, but it looks more confusing than necessary to me.

What about providing a "distance travelled by the sound wave" in meters and then converting that into the observation time in seconds?

Reply:
OK, no problem.